src/cpp/FastCdr.cpp

```diff
@@ -777,7 +777,7 @@ FastCdr& FastCdr::deserializeStringSequence(
     catch (eprosima::fastcdr::exception::Exception& ex)
     {
         free(sequence_t);
-        sequence_t = NULL;
+        delete [] sequence_t;
```

Comment:
I think you wanted to change the previous line, not this one.

Reply:
Good catch. Will fix.